src/content/docs/game-design/godot/dungeoncrawler/1-player/index.mdx

```diff
@@ -0,0 +1,145 @@
+---
+type: tutorial
+unitTitle: Creating our player
+title: Setting up our player.
+description: Creating our player scene and scripts
+---
+
+import Checklist from '/src/components/tutorial/Checklist.astro';
+import Box from '/src/components/tutorial/Box.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+## Creating the Player
+
+:::note[Godot Documentation]
+Godot Documentation for nodes discussed in this section: 
+
+[Collision Shapes](https://docs.godotengine.org/en/stable/tutorials/physics/collision_shapes_2d.html) [CharacterBody2D](https://docs.godotengine.org/en/stable/classes/class_characterbody2d.html) [AnimatedSprite2D](https://docs.godotengine.org/en/stable/classes/class_animatedsprite2d.html) [Area2D](https://docs.godotengine.org/en/stable/classes/class_area2d.html)
+:::
+
+
+Great! Let's start by making a basic version of our player character that will let us move around. We'll worry about more complicated things like attacking later.
+
+
+<Steps>
+
+1. Start by creating a new 2D Scene, with a **CharacterBody2D** as the root node. call it something like **Player**
```

Comment:
Instead of saying call it "like" Player jsut say call it Player, don't have things ambiguous

Reply:
Very good call, best to be clear. Fixed.